tests/integration/kubernetes/gha-run.sh

```diff
@@ -415,6 +415,11 @@ function cleanup() {
 }
 
 function deploy_snapshotter() {
+	if [[ "${KATA_HYPERVISOR}" == "qemu-tdx" ]]; then
+	       echo "[Skip] ${SNAPSHOTTER} is pre-installed in the TEE machine"
+
```

Comment:
Hi @fidencio !

I think you want it to return here.

Reply:
When I added the log I mistakenly removed the return, sorry.